### PR TITLE
add ftp helper

### PR DESCRIPTION
**Pull request by vchepkov**
This adds ability to enable a connection tracker helper and provides typical ftp rules

### CODE REVIEW

**Comment by duritong** [comment]
except the missing docstrings and thus the failing tests, this looks good to me.

File: manifests/helper.pp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# manage a helper
```
Comment (bastelfreak) [inline manifests/helper.pp:1]: please add puppet-strings style documentation for the parameters.

Comment (vchepkov) [reply manifests/helper.pp]: Done. I rebased on top of master and changed commit message to credit @jay7x , let me know if there is a better way

**Review by jay7x — APPROVED** [review]

File: files/config/puppet-inet-filter.nft [hunk 2]
```diff
@@ -1 +1,2 @@
+  include "inet-filter-helper-*.nft"
```
Comment (duritong) [inline files/config/puppet-inet-filter.nft:1]: Can't recall it from the top of my head, but I guess that is fine, even if that expands to 0 files - if no helper is defined?

Comment (vchepkov) [reply files/config/puppet-inet-filter.nft:1]: correct, similar to other optional resources:
```
# grep include /etc/nftables/puppet.nft 
include "file-*.nft"
include "custom-*.nft"
include "inet-filter.nft"

```

**Comment by kenyon** [comment]
The way to credit co-authors is to use `Co-authored-by: NAME <NAME@EXAMPLE.COM>`

See https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Comment by vchepkov** [comment]
> The way to credit co-authors is to use `Co-authored-by: NAME <NAME@EXAMPLE.COM>`
> 
> See https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Thank you, amended

File: manifests/helper.pp [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+#  }
+#
+# @param content
+#   Conntrack helper definition.
+# @param table
+#   The name of the table to add this helper to.
+# @param helper
+#   The symbolic name for the helper.
+define nftables::helper (
+  # lint:ignore:parameter_documentation
```
Comment (ekohl) [inline manifests/helper.pp:15]: Why is this needed? Looks like you have documented them.

Comment (vchepkov) [reply manifests/helper.pp:15]: I copied it from an existing define. let me drop it

Comment (ekohl) [reply manifests/helper.pp:15]: Did you forget to push your changes?

Comment (vchepkov) [reply manifests/helper.pp]: it seems that way. sigh. Hopefully that time is a charm
